### PR TITLE
Update cloud-native-security.md

### PR DESCRIPTION
**Pull request by Sagar2366**

### CODE REVIEW

File: content/en/docs/concepts/security/cloud-native-security.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ The Runtime phase comprises three critical areas: [compute](#protection-runtime-
 ### Runtime protection: access {#protection-runtime-access}
 
 The Kubernetes API is what makes your cluster work. Protecting this API is key
-to providing effective cluster security.
+to provide effective cluster security.
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/security/cloud-native-security.md:94]: Original statement works fine.

File: content/en/docs/concepts/security/cloud-native-security.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ You can enforce measures from the _distribute_ phase, such as verifying the
 cryptographic identity of container image artifacts.
 
 When you deploy Kubernetes, you also set the foundation for your
-applications' runtime environment: a Kubernetes cluster (or
+applications runtime environment: a Kubernetes cluster (or
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/security/cloud-native-security.md:80]: Original statement is grammatically correct.

File: content/en/docs/concepts/security/cloud-native-security.md [hunk 3]
```diff
@@ -105,7 +105,7 @@ components.
 
 Kubernetes uses TLS to protect API traffic; make sure to deploy the cluster using
 TLS (including for traffic between nodes and the control plane), and protect the
-encryption keys. If you use Kubernetes' own API for
+encryption keys. If you use Kubernetes own API for
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/security/cloud-native-security.md:108]: The original statement is grammatically correct.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

**Comment by sftim** [comment]
/sig security

Please write a description for this pull request. Part of the role of a reviewer is to check that the effect of merging the PR matches the description.

**Comment by Ritikaa96** [comment]
Hey @Sagar2366 are you able to address comments from @sftim and @divya-mohan0209 ? Please add description to the PR as asked by sftim.

**Pull request closed without merging**